File: example/scalalib/testing/3-integration-suite/qux/test/src/QuxTests.scala
package qux
import utest.*
object QuxTests extends TestSuite {
  def tests = Tests {
    test("hello") {
      val result = Qux.hello()
      assert(result.startsWith("Hello"))
      result
    }
    test("world") {
      val result = Qux.hello()
      assert(result.endsWith("World"))
      result
    }
  }
}

File: example/javalib/config/2-testing/test/src/qux/QuxTests.java
package qux;

import static org.junit.jupiter.api.Assertions.assertTrue;

import org.junit.jupiter.api.Test;

public class QuxTests {

  @Test
  public void hello() {
    String result = Qux.hello();
    assertTrue(result.startsWith("Hello"));
  }

  @Test
  public void world() {
    String result = Qux.hello();
    assertTrue(result.endsWith("World"));
  }
}
